### PR TITLE
Feature/e UI 3232 wa integration/revert and fix tabs

### PR DESCRIPTION
**Pull request by connorpgpmcelroy**
### JIRA link (if applicable) ###
N/A

### Change description ###
Fixing tabs differently to avoid issues when url changes exacted with launchDarkly

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: api/workAllocation/constants/actions.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ const ASSIGN = {
 };
 const CANCEL = {
   id: 'cancel',
-  title: 'Cancel task'
+  title: 'Cancel task',
```
Comment (paul-graham) [inline api/workAllocation/constants/actions.ts:7]: What are these extra commas for? Auto-formatter or something?

Comment (connorpgpmcelroy) [reply api/workAllocation/constants/actions.ts:7]: Linting errors I was getting. These are occurring on wa-integration branch as well so will fix that I believe

File: src/app/app-utils.ts [hunk 2]
```diff
@@ -66,14 +66,62 @@ export class AppUtils {
...
+      checkHrefs = checkItem.href === '/tasks' ? ['/tasks/list', '/tasks/available'] : [checkItem.href];
+      fullUrl = AppUtils.isFullUrl(checkItem.href, currentUrl);
+      if (fullUrl) {
+        break;
+      }
+      if (checkItem.href === '/cases') {
+        // if cases we need an equivalence to stop confusion in tab selection
+        continue;
+      }
+      // if the href partly matches, find the largest tab name for which the url partly matches
```
Comment (paul-graham) [inline src/app/app-utils.ts:103]: I assume you mean the best match for the `href`, not `tab name`?

Comment (connorpgpmcelroy) [reply src/app/app-utils.ts:103]: yes, I can change that

File: api/workAllocation/constants/actions.ts (hunk 1 above)
Comment (udaydenduluri33) [reply api/workAllocation/constants/actions.ts:7]: Yeah the Lint is dumb. Unfortunately its required

Comment (paul-graham) [reply api/workAllocation/constants/actions.ts:7]: A trailing comma is _required_? That's absurd!

**Review by paul-graham — APPROVED** [review]

**Review by udaydenduluri33 — APPROVED** [review]
LGTMc

**Pull request merged**